src/controllers/bulk/bulkcontroller.h

```diff
@@ -56,6 +56,7 @@ class BulkController : public Controller {
 
     virtual bool savePreset(const QString fileName) const;
 
+    virtual void visit(const KeyboardControllerPreset* preset);
```

Comment:
I don't think I quite understand what you mean by naming these visit functions explicit. Could you clarify please?